hooks/tflint.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+
+# OSX GUI apps do not pick up environment variables the same way as Terminal apps and there are no easy solutions,
+# especially as Apple changes the GUI app behavior every release (see https://stackoverflow.com/q/135688/483528). As a
+# workaround to allow GitHub Desktop to work, add this (hopefully harmless) setting here.
+export PATH=$PATH:/usr/local/bin
+
+for file in "$@"; do
+  tflint `dirname $file`
```

Comment:
Same issue as the terraform validate PR

Reply:
`tflint` can work with files and directories. IMHO, if the tool may work with files, we should leverage on the `pre-commit`'s parallelization default.

Reply:
Sounds good. The only reason why I highlighted that was because you were using `dirname`.